integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/metadata/MetadataConverter.java

```diff
@@ -49,6 +50,23 @@ private CertificateUnit convertUnit(CertificateMetadataDTO metadataDTO) {
     return CertificateUnit.builder()
         .id(metadataDTO.getUnit().getUnitId())
         .name(metadataDTO.getUnit().getUnitName())
+        .address(metadataDTO.getUnit().getAddress())
+        .zipCode(metadataDTO.getUnit().getZipCode())
+        .city(metadataDTO.getUnit().getCity())
+        .phoneNumber(metadataDTO.getUnit().getPhoneNumber())
+        .email(metadataDTO.getUnit().getEmail())
+        .build();
+  }
+
+  CertificateUnit convertCareUnit(CertificateMetadataDTO metadataDTO) {
```

Comment:
Could we change the signature of convertUnit method so that it recieves the getUnit() object and use it for careUnit aswell?